dir.props

```diff
@@ -90,6 +87,9 @@
     <CoreFxExpectedPrerelease>rc3-24015-00</CoreFxExpectedPrerelease>
   </PropertyGroup>
 
+  <!-- Import packaging props -->
+  <Import Project="$(MSBuildThisFileDirectory)Packaging.props"/>
+
```

Comment:
Does this import need to be dropped lower in the ordering, if so does it also need to be done in corefx?